library/src/test/java/org/apache/apex/malhar/lib/state/managed/ManagedTimeStateImplTest.java

```diff
@@ -81,6 +83,7 @@ public void testAsyncGetFromReaders() throws IOException, ExecutionException, In
     Slice zero = ManagedStateTestUtils.getSliceFor("0");
     long time = System.currentTimeMillis();
 
+    DefaultBucket.setDisableBloomFilterByDefault(true);
```

Comment:
the bloom filter works only go through put() and get() method of bucket. But this unit test case write data direct data from file instead of go through put(), so the bloom filter didn't keep the flags and get() return null due to filtered by Bloom Filter.